src/main/java/org/sahagin/runlib/external/adapter/fluentlenium/FluentLeniumAdapter.java

```diff
@@ -26,16 +26,16 @@ public String getName() {
     }
 
     // can set null
-    public static void setAdapter(Fluent fluent) {
+    public static void setAdapter(FluentDriver fluent) {
         AdapterContainer container = AdapterContainer.globalInstance();
```

Comment:
yes.
the architecture of FluentLenium was drastically changed from v.0.13.x to v.1.x.x.
`Fluent` on v.0.13.x is proxy of WebDriver. so I think `FluentDriver` on v.1.x.x is suitable for substitute.
this breaks backward compatibility.
please notify about that on changelog.